client/config/crd/cbt.storage.k8s.io_snapshotmetadataservices.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.15.0
+  name: snapshotmetadataservices.cbt.storage.k8s.io
+spec:
+  group: cbt.storage.k8s.io
+  names:
+    kind: SnapshotMetadataService
+    listKind: SnapshotMetadataServiceList
+    plural: snapshotmetadataservices
+    shortNames:
+    - sms
+    singular: snapshotmetadataservice
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: |-
+          SnapshotMetadataService is the Schema for the snapshotmetadataservices API
```

Comment:
Missing period.

> SnapshotMetadataService is the Schema for the snapshotmetadataservices API

What exactly does this sentence mean?

Reply:
This line is added when CRD is generated with `controller-gen`. I'll check if we can modify it